### PR TITLE
Fix `TOC` on Desktop & Mobile

### PR DESCRIPTION
**Pull request by ItzNotABug**
## What does this PR do?

As per the discussions, TOC should not show on desktops. Also fixes broken on mobile.

## Test Plan

Manual.

## Related PRs and Issues

N/A.

### Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?

Yes.

### CODE REVIEW

File: src/routes/assets/+page.svelte [hunk 1]
```diff
@@ -64,22 +66,27 @@
...
+                                <span class="icon-menu-alt-4" aria-hidden="true" />
+                            </span>
+                        </button>
+                    </div>
                 </header>
-                <TocNav />
+                <TocNav bind:showToc />
                 <main class="web-grid-120-1fr-auto-main /web-is-mobile-closed" id="main">
-                    <div class="web-content">
+                    <div class="web-content" class:web-is-mobile-closed={showToc && !showToc}>
```
Comment (loks0n) [inline src/routes/assets/+page.svelte:89]: `{showToc && !showToc}` should always evaluate false right?

Comment (ItzNotABug) [reply src/routes/assets/+page.svelte:89]: I think we might not even need this class here. This PR still requires further fixes anyways.

File: src/lib/components/TocNav.svelte [hunk 2]
```diff
@@ -12,94 +13,194 @@
...
+            easing,
+            css: (t: number) => {
+                return `
+                    opacity: ${t};
+                    height: ${t * initialHeight}px;
+                    overflow: hidden;
+                `;
+            }
+        };
+    }
```
Comment (ernstmul) [inline src/lib/components/TocNav.svelte:39]: Can we also do this animation with CSS? I think that would better fit the direction we're moving in. Wdyt @thejessewinton?

File: src/markdoc/layouts/Policy.svelte [hunk 3]
```diff
@@ -107,7 +98,20 @@
         opacity: 0;
     }
 
-    .toc-btn {
-        transition: translate 0.3s ease;
+    @media (max-width: 768px) {
+        .container {
+            padding-left: 0 !important;
+            padding-right: 0 !important;
```
Comment (ernstmul) [inline src/markdoc/layouts/Policy.svelte:104]: Are these `!important`'s really needed?

File: src/routes/assets/+page.svelte [hunk 4]
```diff
@@ -628,4 +622,21 @@
             }
         }
     }
+
+    @media (max-width: 768px) {
+        .container {
+            padding-left: 0 !important;
+            padding-right: 0 !important;
```
Comment (ernstmul) [inline src/routes/assets/+page.svelte:629]: Same question about important

File: src/lib/components/TocNav.svelte (hunk 2 above)
Comment (ItzNotABug) [reply src/lib/components/TocNav.svelte:39]: Not sure if we probably can (due to `t * initialHeight`, but correct me if I am wrong), but I think I've seen a similar use-case somewhere around the repo. Lmk what we decide.

File: src/markdoc/layouts/Policy.svelte (hunk 3 above)
Comment (ItzNotABug) [reply src/markdoc/layouts/Policy.svelte]: I tested again, not needed. removed.

File: src/routes/assets/+page.svelte (hunk 4 above)
Comment (ItzNotABug) [reply src/routes/assets/+page.svelte]: Addressed.

File: src/lib/components/TocNav.svelte (hunk 2 above)
Comment (ItzNotABug) [reply src/lib/components/TocNav.svelte:39]: ping @ernstmul @thejessewinton.

**Review by ernstmul — APPROVED** [review]

**Pull request merged**